src/Type.cpp

```diff
@@ -519,6 +539,30 @@ Type::get_all_ok_struct_union_types(vector<Type *> &ok_types, bool no_const, boo
 	}
 }
 
+bool 
+Type::if_struct_will_have_assign_ops()
+{
+	bool hasAssignOps = false;
+	// randomly choose if the struct will have assign operators (for C++):
+	if (CGOptions::lang_cpp()){
+		hasAssignOps = rnd_flipcoin(RegularVolatileProb);
+	}
+	return hasAssignOps;
+}
```

Comment:
Perhaps the code can be simplified to? 

    if (!CGOptions::lang_cpp())
        return false;
    return rnd_flipcoin(RegularVolatileProb);